src/main/resources/fxml/home/directionMessage.fxml

```diff
@@ -12,11 +12,11 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.scene.shape.Polygon?>
 
-<VBox alignment="TOP_CENTER" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="952.0" prefWidth="480.0" spacing="5.0" stylesheets="@/theme.css" xmlns="http://javafx.com/javafx/8.0.171" xmlns:fx="http://javafx.com/fxml/1" fx:controller="home.DirectionsController">
+<VBox alignment="TOP_CENTER" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="952.0" prefWidth="480.0" spacing="5.0" stylesheets="@../../theme.css" xmlns="http://javafx.com/javafx/8.0.171" xmlns:fx="http://javafx.com/fxml/1" fx:controller="home.DirectionsController">
     <children>
         <HBox alignment="TOP_CENTER">
             <children>
-                <JFXButton fx:id="unitSwitch_btn" minWidth="-Infinity" onAction="#setUnits" prefHeight="56.0" prefWidth="106.0" styleClass="confirm-button" text="Meters" />
+                <JFXButton fx:id="unitSwitch_btn" minWidth="-Infinity" onAction="#setUnits" prefHeight="56.0" prefWidth="56.0" styleClass="confirm-button" text="M" />
```

Comment:
If we're abbreviating, shouldn't the 'M' be lowercase?

Reply:
It's uppercase M in the code itself, I only changed it here to have the FXML correctly reflect what it looks like in app